projects/demo/src/modules/components/editor/starter/editor-starter.template.html

```diff
@@ -132,6 +132,20 @@ <h2>TUI_EDITOR_OPTIONS</h2>
                 <strong>Description of the available configurations:</strong>
             </p>
             <dl>
+                <dt>
+                    <code>emitUpdateTime</code>
```

Comment:
```suggestion
                    <code> debounceTime </code>
```

I find it more clear and obvious naming.